integration-tests/tests/rollbackApp_test.go

```diff
@@ -0,0 +1,110 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+// +build !excludeintegration
+
+/*
+ * Copyright (C) 2015, 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package tests
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/snapcore/snapd/integration-tests/testutils/cli"
+	"github.com/snapcore/snapd/integration-tests/testutils/common"
+	"github.com/snapcore/snapd/integration-tests/testutils/config"
+	"github.com/snapcore/snapd/integration-tests/testutils/refresh"
+	"github.com/snapcore/snapd/integration-tests/testutils/store"
+	"github.com/snapcore/snapd/testutil"
+
+	"gopkg.in/check.v1"
+)
+
+var _ = check.Suite(&rollbackAppSuite{})
+
+type rollbackAppSuite struct {
+	common.SnappySuite
+}
+
+func (s *rollbackAppSuite) TestInstallUpdateRollback(c *check.C) {
```

Comment:
This needs to be moved to spread, but let's not block on it. I see it depends on the fake store, which will need a bit of churn for us to establish the pattern. So a follow up seems more appropriate.